src/index.js

```diff
@@ -23,33 +23,43 @@ import {
 
 import { initCSSPolyfill } from "./scroll-timeline-css"
 
-initCSSPolyfill();
+export function initPolyfill() {
```

Comment:
Why do we need to export this if calling it directly below?

Reply:
Agree that this is currently unnecessary.  
There could be an argument that folks might want to know if the polyfill is active or just using the native browser implementation (so the return true at the top of the function might be interesting), but folks can't call that function safely now (since they would need to call it a second time, and it doesn't handle that gracefully.)

Pulling this out.